uavcan_data_types/cvra/actuator/20150.Feedback.uavcan

```diff
@@ -0,0 +1,7 @@
+#
+# Feedback coming from actuator-firmware nodes
+#
+
+float16[2] pressures       # [Pascal]
```

Comment:
In other message definitions we use singular for fields, even in case of an array.
Personally I prefer singular because eg. `feedback.analog_input[1]` looks nicer.

Reply:
done